### PR TITLE
Add steps to try out pull_from_upstream

### PR DESCRIPTION
**Pull request by nforro**

### CODE REVIEW

File: docs/fedora-releases-guide.md [hunk 1]
```diff
@@ -138,6 +138,27 @@ configuration key.
 
 ![Dashboard view for pull_from_upstream](img/fedora-releases-guide/pull-from-upstream-dashboard.png)
 
+#### Trying it out
+If you are interested in this functionality and want to try it out, you can trigger the job from a pull request.
+
+If there is a new release pending for your package ([release-monitoring.org](https://release-monitoring.org/) bugzilla has been opened but no rebase done in dist-git yet), do the following:
+
```
Comment (lbarcziova) [inline docs/fedora-releases-guide.md:145]:
```suggestion
If there is a new release pending for your package (Bugzilla created by [release-monitoring.org](https://release-monitoring.org/) has been opened, but no rebase has been done in dist-git yet), do the following:

```

File: docs/fedora-releases-guide.md [hunk 2]
```diff
@@ -138,6 +138,27 @@ configuration key.
...
+- create a `rawhide`-based pull request with Packit configuration defining the [`pull_from_upstream` job](/docs/configuration/downstream/pull_from_upstream)
+- comment `/packit pull-from-upstream --with-pr-config` on the pull request
+- check the [dashboard](https://dashboard.packit.dev/jobs/pull-from-upstreams)
+- if everything went well, review the pull request(s) created by Packit
```
Comment (lbarcziova) [inline docs/fedora-releases-guide.md:149]:
```suggestion
- if everything went well, review the pull request(s) in your dist-git repository created by Packit
```

File: docs/fedora-releases-guide.md [hunk 3]
```diff
@@ -138,6 +138,27 @@ configuration key.
...
+- if you are happy with the results, merge your pull request and comment `/packit pull-from-upstream` to run the job for real (existing Packit-created pull requests will be updated)
```
Comment (lbarcziova) [inline docs/fedora-releases-guide.md:150]: > comment `/packit pull-from-upstream` to run the job for real (existing Packit-created pull requests will be updated)

this should not be needed, or am I missing something?

File: docs/fedora-releases-guide.md [hunk 4]
```diff
@@ -138,6 +138,27 @@ configuration key.
...
+
+If there is no pending release and your package has been rebased at least once in the past, you can still try the job using a private branch:
```
Comment (lbarcziova) [inline docs/fedora-releases-guide.md:152]:
```suggestion
If there is no pending release and your package has been rebased at least once in the past, you can still try the job using a new testing branch:
```

File: docs/fedora-releases-guide.md [hunk 5]
```diff
@@ -138,6 +138,27 @@ configuration key.
...
+
+- create a branch pointing to a commit before the last rebase, name it e.g. `packit-test` and push it (directly to dist-git, not to your fork)
+- create a `rawhide`-based pull request with Packit configuration defining the [`pull_from_upstream` job](/docs/configuration/downstream/pull_from_upstream)
+- in the configuration, set the `dist_git_branches` attribute of the `pull_from_upstream` job to the name of the private branch
```
Comment (lbarcziova) [inline docs/fedora-releases-guide.md:156]:
```suggestion
- in the configuration, set the `dist_git_branches` option of the `pull_from_upstream` job to the name of the testing branch
```

**Review by lbarcziova — COMMENTED** [review]
really nice! Just few comments

File: docs/fedora-releases-guide.md (hunk 3 above)
Comment (nforro) [reply docs/fedora-releases-guide.md:150]: True, not really needed, I'll remove it.

File: docs/fedora-releases-guide.md (hunk 4 above)
Comment (nforro) [reply docs/fedora-releases-guide.md:152]: I believe "private branch" is a coined term, at least for RHEL package maintainers. However, I can't find any mention of it in Fedora documentation. So, let's go with your suggestion.

File: docs/fedora-releases-guide.md (hunk 1 above)
Comment (nforro) [reply docs/fedora-releases-guide.md:145]: I don't like the "conflict" between created and opened here, how about "bugzilla has been opened by release-monitoring.org"?

Comment (lbarcziova) [reply docs/fedora-releases-guide.md:145]: I missed that, sure, that makes sense

**Review by lbarcziova — APPROVED** [review]
:rocket:

**Comment by nforro** [comment]
regate

**Pull request merged**